DEV_SETUP_MAC.md

```diff
@@ -124,6 +124,12 @@
   export LDFLAGS="-L/opt/homebrew/opt/libpq/lib"
   pip install psycopg2-binary
   ```
+  
+  Or try: ([reference](https://rogulski.it/blog/install-psycopg2-on-apple-m1/))
```

Comment:
Might be worth specifying what version of Mac OS this definitely works on (whatever you're using)

Reply:
resolved in commit [adb7485](https://github.com/dimagi/commcare-hq/pull/32509/commits/adb748558e25a491aedd48a9ce45c403a7e428fd)